
# Statistical Analysis of High-Throughput Genomic and Transcriptomic Data
Fall/Herbst-semester 2025

## Lectures
Mondays 9.00-9.45 (Y27-H-46), 10.00-10.45 (Y27-H-46)

## Exercises
Monday 11.00-11.45

## Lecturers

[Dr. Hubert Rehrauer](http://www.fgcz.ch/the-center/people/rehrauer.html), Group Leader of Genome Informatics at [FGCZ](http://www.fgcz.ch/)

[Prof. Dr. Mark Robinson](https://robinsonlabuzh.github.io/), Professor of Statistical Genomics, [DMLS](https://www.mls.uzh.ch/en.html), UZH

[tba](https://www.uzh.ch], Title, [Affiliation](https://ethz.ch/), UZH/ETHZ

## Schedule

| Date  | Lecturer | Topic | Exercise | JC1 | JC2 |
| --- | --- | --- | --- | --- | --- |
| 15.09.2025  | Mark+Hubert | admin; mol. bio. basics | quarto; git(hub) | | |
| 22.09.2025  | Mark | interactive technology/statistics session  | group exercise: technology PR | | |
| 29.09.2025  | Hubert | NGS intro; exploratory data analysis | EDA in R | | |
| 06.10.2025  | Hubert | mapping  | Rsubread | | |
| 13.10.2025  | Mark  | limma + friends | linear model simulation + design matrices | | |
| 20.10.2025  | Hubert | RNA-seq quantification | RSEM  | X | X | 
| 27.10.2025  | Mark | edgeR+friends 1 | basic edgeR/voom | [Analysis of multi-condition single-cell data with latent embedding multivariate regression](https://www.nature.com/articles/s41588-024-01996-0) (AM, AL, AC)| [GECSI: Large-scale chromatin state imputation from gene expression](https://www.biorxiv.org/content/10.1101/2025.10.02.679828v1) (ES, GC, NM) |
| 03.11.2025  | Mark | edgeR+friends 2 | advanced edgeR/voom | [Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles](https://www.pnas.org/doi/10.1073/pnas.0506580102) (NH, HQ) | [voom: precision weights unlock linear model analysis tools for RNA-seq read counts](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r29) (TH, JB)|
| 10.11.2025  | Mark | hands-on session #1: RNA-seq |  FASTQC/Salmon/etc. | [Differential abundance testing on single-cell data using k-nearest neighbor graphs](https://www.nature.com/articles/s41587-021-01033-z) (XX,TK,TB) | [spicyR: spatial analysis of in situ cytometry data in R](https://academic.oup.com/bioinformatics/article/38/11/3099/6570585) (SI, JH) |
| 17.11.2025  | Hubert | single-cell 1: preprocessing, dim. red., clustering | clustering | [Application of Weighted Gene Co-expression Network Analysis  for Data from Paired Design](https://www.nature.com/articles/s41598-017-18705-z) (TR, FD, FH) | [TISSUE: uncertainty-calibrated prediction of single-cell spatial transcriptomics improves downstream analyses](https://www.nature.com/articles/s41592-024-02184-y) (AB, MF, EK) |
24.11.2025  | Mark | single-cell 2: clustering, marker gene DE | marker gene DE | [hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data](https://www.cell.com/cell-reports-methods/fulltext/S2667-2375(23)00127-3) (CP, CK) | X |
| 01.12.2025  | tba | hands-on session #2: single-cell RNA-seq | full scRNA-seq pipeline | [CHOIR improves significance-based detection of cell types and states from single-cell data](https://www.nature.com/articles/s41588-025-02148-8) (MK, PB) | [Redefining CpG islands using hidden Markov models](https://doi.org/10.1093/biostatistics/kxq005) (PW, SB)  |
| 08.12.2025  | Mark | spatial omics 1 | spatial statistics | X | X |
| 15.12.2025  | Mark | spatial omics 2 | structures, DSP | [Cell type-specific inference of differential expression in spatial transcriptomics](https://www.nature.com/articles/s41592-022-01575-3) (DS, DF) | X |


## Course material

Assuming you have git installed locally, you can check out the entire set of course materials with the following command (from command line):
```
git clone https://github.com/sta426hs2025/material.git
```
and get updates by running `git pull` at any later time in the same directory.

Alternatively, to retrieve a ZIP file of the repository, you can click on the (green) 'Code' button (top right of main panel) and then click 'Download ZIP'.

